dist/images/start-db.sh

```diff
@@ -6,7 +6,7 @@ DEBUG_OPT="--ovn-northd-wrapper=$DEBUG_WRAPPER --ovsdb-nb-wrapper=$DEBUG_WRAPPER
 
 echo "PROBE_INTERVAL is set to $PROBE_INTERVAL"
 echo "OVN_LEADER_PROBE_INTERVAL is set to $OVN_LEADER_PROBE_INTERVAL"
-
+echo "OVN_NORTHD_N_THREADS is set to $OVN_NORTHD_N_THREADS"
```

Comment:
we need add a default value in case user only update the image

Reply:
```OVN_NORTHD_N_THREADS=${OVN_NORTHD_N_THREADS:-1}```

Reply:
ok